pyro/distributions/zero_inflated.py

```diff
@@ -70,17 +71,29 @@ def log_prob(self, value):
         if self._validate_args:
             self._validate_sample(value)
 
+        support = self.support
+        epsilon = abs(torch.finfo(value.dtype).eps)
```

Comment:
Just curious, what does the `abs(-)` here do?

Reply:
I just want to make sure that `epsilon` is positive. Say, for example for a `Beta` distribution, the bounds are (0,1). So if `value=0` and `epsilon<0` then we have a negative `value`.

Reply:
Well `finfo(...).eps` is always positive, but I guess an `abs(-)` doesn't hurt